packages/react-vis/tests/components/xy-plot-tests.js

```diff
@@ -297,21 +237,16 @@ test('Trigger all onParentMouse handlers on Series components', t => {
     .at(0)
     .simulate('touchmove');
 
-  t.end();
+  expect(onParentMouseHandler).toHaveBeenCalledTimes(14);
```

Comment:
this assertion is not great but I wanted to migrate to jest by keeping the tests identical if possible and not bring any opinionated change